src/App.jsx

```diff
@@ -14,45 +17,103 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+function getPreparedGoods(goods, { sortField, reverseArray }) {
+  const visibleGoods = [...goods];
+
+  if (sortField) {
+    visibleGoods.sort((first, second) => {
+      switch (sortField) {
+        case SORT_BY.ALPHABET:
+          return first.localeCompare(second);
+
+        case SORT_BY.LENGTH:
+          return first.length - second.length;
+
+        default:
+          return visibleGoods;
+      }
+    });
+  }
+
+  if (reverseArray) {
+    visibleGoods.reverse();
+  }
+
+  return visibleGoods;
+}
+
+const SORT_BY = {
+  ALPHABET: 'alphabet',
+  LENGTH: 'length',
+};
```

Comment:
better to name object as always in camelCase and constants inside in UPPER_SNAKE, or just create two constants, it is not much